tests/test_pipeline/test_pipeline.py

```diff
@@ -336,6 +336,23 @@ def test_forecast_prediction_interval_not_builtin_with_nans_error(example_tsds,
         _ = pipeline.forecast(prediction_interval=True, quantiles=[0.025, 0.975])
 
 
+@pytest.mark.filterwarnings("ignore: There are NaNs in target on time span from .* to .*")
+@pytest.mark.parametrize("model", (MovingAverageModel(),))
+@pytest.mark.parametrize("stride", (1, 4, 6))
+def test_add_forecast_borders_overlapping_timestamps(example_tsds, model, stride):
+    example_tsds.df.loc[example_tsds.index[-20:-1], pd.IndexSlice["segment_1", "target"]] = None
```

Comment:
Why are we adding missing values here?

Reply:
Just to ensure that indices of missing values are not ignored